src/NServiceBus.AcceptanceTests/Sagas/When_correlated_property_value_is_changed.cs

```diff
@@ -21,10 +21,9 @@ await Scenario.Define<Context>()
                         {
                             DataId = Guid.NewGuid()
                         })))
-                    .Done(c => c.FailedMessages.Any())
+                    .Done(c => c.ModifiedCorrelationProperty)
```

Comment:
I think it should be no issue, as the att framework should wait for "incomplete" message to be processed (to either succeed or fail).